features/vendor_views_closed_auction.feature

```diff
@@ -13,6 +13,8 @@ Feature: Vendor views closed auctions
     And I should see when the auction started
     And I should see when the auction ended
     And there should be meta tags for the closed auction
+    And an email notification is sent to the winning bidder
+    And I should receive an email notifying me that I won
```

Comment:
hah yeah maybe we create a new feature spec for daily cron job-related stuff and in those tests we use time traveling steps? again, not sure how we can get clockwork running in cucumber but maybe not so hard...